--- ./Makefile	2013-05-27 22:48:43.084868701 +0200
+++ ./Makefile.reproducible	2015-05-14 15:53:48.928027622 +0200
@@ -12,7 +12,7 @@
 all: $(OBJECTS) $(NAME)
 
 $(NAME): $(SOURCES) $(OBJECTS)
-	$(CC) -o $(NAME) $(CPPFLAGS) $(CFLAGS) $(NAME).c $(OBJECTS) $(LDFLAGS) -DPXZ_BUILD_DATE=\"`date +%Y%m%d`\" -DPXZ_VERSION=\"$(VERSION)\"
+	$(CC) -o $(NAME) $(CPPFLAGS) $(CFLAGS) $(NAME).c $(OBJECTS) $(LDFLAGS) -DPXZ_BUILD_DATE=\"$(shell dpkg-parsechangelog -S date|date -u +%Y%m%d)\" -DPXZ_VERSION=\"$(VERSION)\"
 
 clean:
 	rm -f *.o $(NAME)
